compiler/data/class-members.cpp

```diff
@@ -96,8 +97,8 @@ ClassMemberInstanceField::ClassMemberInstanceField(ClassPtr klass, VertexAdaptor
   modifiers(modifiers),
   root(root),
   phpdoc(phpdoc),
-  type_hint(type_hint) {
-
+  type_hint(type_hint),
+  kphp_json_tags(new kphp_json::FieldTags) {
```

Comment:
It seems not logical that this field is non-null for all field. It's better to allocate memory only when a field has @kphp-json tag (and/or, if necessary, if a class has @kphp-json tag)
See below.